mscore/shortcut.cpp

```diff
@@ -2591,6 +2591,20 @@ Shortcut Shortcut::_sc[] = {
          QT_TRANSLATE_NOOP("action","Zoom to 100%"),
          QT_TRANSLATE_NOOP("action","Zoom to 100%")
          },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT | STATE_PLAY,
+         "zoom-page-width",
+         QT_TRANSLATE_NOOP("action","Zoom to page-width / previous magnification level"),
+         QT_TRANSLATE_NOOP("action","Zoom to page-width / previous magnification level")
+         },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT | STATE_PLAY,
+         "zoom100",
+         QT_TRANSLATE_NOOP("action","Zoom to 100%"),
+         QT_TRANSLATE_NOOP("action","Zoom to 100%")
+         },
```

Comment:
This block should probably be removed, `zoom100` shortcut is already defined a few lines above.

Reply:
Total accident.